test/test.py

```diff
@@ -42,123 +42,228 @@ def outdir(test_method_dir: str) -> str:
+#
+#         size_kb = self.run_command(test_name, outfile, command_str)
+#         self.assertGreaterEqual(size_kb, filesize_threshold_kb)
+#
+#     def test_cli_4_merge(self):
+#         """Test CLI: merge"""
+#         test_name = 'test_cli_4_merge'
+#         outfile = 'merged_loinc.owl'
+#         filesize_threshold_kb = 500  # semi-arbitrary for now
+#         command_str = \
+#             '{} merge ' \
+#             '--owl-directory ./test/static/test_cli_4_merge/input/ ' \
+#             '--output {}'
+#
+#         # Setup
+#         input_dir = './test/static/test_cli_4_merge/input/'
+#         inputs = [os.path.join(*[TEST_STATIC_DIR] + x) for x in [
+#             ['test_cli_1_parts', 'output', 'part_ontology.owl'],
+#             ['test_cli_2_codes', 'output', 'code_classes.owl'],
+#             ['test_cli_3_composed', 'output', 'composed_component_classes.owl']
+#         ]]
+#         # todo: Fix: PermissionError: [Errno 1] Operation not permitted: './test/static/test_cli_merge/input/'
+#         # if os.path.exists(input_dir):
+#         #     os.remove(input_dir)  # just in case last test failed and something goes wrong in this test
+#         Path(input_dir).mkdir(parents=True, exist_ok=True)
+#         for x in inputs:
+#             shutil.copy(x, input_dir)
+#
+#         # Run test
+#         size_kb = self.run_command(test_name, outfile, command_str)
+#         self.assertGreaterEqual(size_kb, filesize_threshold_kb)
+#
+#         # Tearown
+#         # todo: Fix: PermissionError: [Errno 1] Operation not permitted: './test/static/test_cli_merge/input/'
+#         # os.remove(input_dir)
+#
+#     def test_cli_5_reason(self):
+#         """Test CLI: reason"""
+#         test_name = 'test_cli_5_reason'
+#         outfile = 'merged_reasoned_loinc.owl'
+#         filesize_threshold_kb = 500  # semi-arbitrary for now
+#         command_str = \
+#             '{} reason ' \
+#             '--merged-owl ./test/static/test_cli_4_merge/output/merged_loinc.owl ' \
+#             '--owl-reasoner elk ' \
+#             '--output {}'
+#
+#         size_kb = self.run_command(test_name, outfile, command_str)
+#         self.assertGreaterEqual(size_kb, filesize_threshold_kb)
 
-    cli_command_base = 'python comp_loinc/build.py'
-
-    def run_command(self, test_name: str, outfile: str, command_str: str) -> float:
-        """Run command for test"""
-        outdir = self.outdir(test_name)
-        Path(outdir).mkdir(parents=True, exist_ok=True)
-        outpath = os.path.join(outdir, outfile)
-        command_str = command_str.format(self.cli_command_base, outpath)
-        command_list = command_str.split(' ')
-        # todo: Fix: PermissionError: [Errno 1] Operation not permitted: ...
-        # if os.path.exists(outpath):
-        #     os.remove(outpath)
-        subprocess.run(command_list, cwd=PROJECT_DIR)
-        size_kb = os.path.getsize(outpath) / 1000
-        return size_kb
+
+class CompLoincPythonAPITests(StaticFileTests):
```

Comment:
New tests I wrote today which call the Python code directly.

@timbrisc yay:
```
make test
python3 -m unittest discover -v
test_python_api_1_parts (test.test.CompLoincPythonAPITests)
Test Python API: parts ... ok
test_python_api_2_codes (test.test.CompLoincPythonAPITests)
Test Python API: codes ... ok
test_python_api_3_composed (test.test.CompLoincPythonAPITests)
Test Python API: composed ... ok
test_python_api_4_merge (test.test.CompLoincPythonAPITests)
Test Python API: merge ... ok
test_python_api_5_reason (test.test.CompLoincPythonAPITests)
Test Python API: reason ... ok

----------------------------------------------------------------------
Ran 5 tests in 162.896s

OK
```

It _may_ be possible that when you run this, you get a `typer` not found error; a result of not `pip` installing correctly. If so, let me know and I can do something to fix. Or, you could simply run the unit tests directly from the command line using `python3 -m unittest discover -v`. 

This won't be a problem in the GitHub action, because I've set it up to do pip installs first.